app/config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
   redisUrl: process.env.REDIS_URL || 'localhost',
 
   // The port redis is running
-  redisPort: process.env.REDIS_PORT || 6379,
+  redisPort: process.env.REDIS_PORT || 6380,
```

Comment:
change this back pretty please

Reply:
By the way you can set this in your .env file. Save you from changing it back and forth when you run it locally